pkg/fftypes/message.go

```diff
@@ -68,30 +68,30 @@ var (
 // MessageHeader contains all fields that contribute to the hash
 // The order of the serialization mut not change, once released
 type MessageHeader struct {
-	ID     *UUID           `ffstruct:"MessageHeader" json:"id,omitempty"`
+	ID     *UUID           `ffstruct:"MessageHeader" json:"id,omitempty" ffexcludeinput:"true"`
 	CID    *UUID           `ffstruct:"MessageHeader" json:"cid,omitempty"`
 	Type   MessageType     `ffstruct:"MessageHeader" json:"type" ffenum:"messagetype"`
-	TxType TransactionType `ffstruct:"MessageHeader" json:"txtype,omitempty"`
+	TxType TransactionType `ffstruct:"MessageHeader" json:"txtype,omitempty" ffexcludeinput:"true"`
```

Comment:
For sending an `unpinned` private message, I believe this needs to be documented for input